modules/react/text-input/stories/TextInput.stories.mdx

```diff
@@ -0,0 +1,113 @@
+import {Meta} from '@storybook/addon-docs/blocks';
+
+import {TextInput} from '@workday/canvas-kit-react/text-input';
+
+import {Alert} from './examples/Alert';
+import {AlertStandalone} from './examples/AlertStandalone';
+import {Default} from './examples/Default';
+import {Disabled} from './examples/Disabled';
+import {Error} from './examples/Error';
+import {ErrorStandalone} from './examples/ErrorStandalone';
+import {Grow} from './examples/Grow';
+import {GrowStandalone} from './examples/GrowStandalone';
+import {LabelPosition} from './examples/LabelPosition';
+import {Placeholder} from './examples/Placeholder';
+import {RefForwarding} from './examples/RefForwarding';
+import {Required} from './examples/Required';
+import {Standalone} from './examples/Standalone';
+
+<Meta title="Components/Inputs/Text Input/React" component={TextInput} />
+
+# Canvas Kit Text Input
+
+Text Inputs allow users to enter words or characters without styling.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-react
+```
+
+## Default Example
+
+Text Input should be used in tandem with Form Field to meet accessibility standards.
```

Comment:
Sure, I've linked "Form Field" throughout the MDX docs to the Form Field Docs tab in Storybook (just the first occurrence of Form Field in each article, though).

I didn't convert "Form Field" to `FormField` yet since it seems like we would then need to convert all component mentions to the `code` format throughout the docs (e.g., "Text Input" to `TextInput`). But maybe we should establish this as a convention now? Alan seems to have done it for [Box](https://workday.github.io/canvas-kit/?path=/docs/labs-common-box--as) and the [Layout components](https://workday.github.io/canvas-kit/?path=/docs/labs-layout-flex--flex-card).

Reply:
Hmm that's a good point. In the context of Storybook, the code formatting makes sense, but maybe not as much on the site...